Build error with `wasi-c` feature

#### how to reproduce
```
cargo build --all-features --release
```
#### Errors
```
error[E0252]: the name `instantiate_wasi_c` is defined multiple times
  --> src/commands/run.rs:23:5
   |
21 | use wasmtime_wasi_c::instantiate_wasi_c;
   |     ----------------------------------- previous import of the value `instantiate_wasi_c` here
22 | #[cfg(feature = "wasi-c")]
23 | use wasmtime_wasi_c::instantiate_wasi_c;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `instantiate_wasi_c` reimported here
   |
   = note: `instantiate_wasi_c` must be defined only once in the value namespace of this module

warning: unused import: `wasmtime_wasi_c::instantiate_wasi_c`
  --> src/commands/run.rs:23:5
   |
23 | use wasmtime_wasi_c::instantiate_wasi_c;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

error[E0061]: this function takes 5 parameters but 4 parameters were supplied
   --> src/commands/run.rs:155:30
    |
155 |                 let handle = instantiate_wasi_c(global_exports, &preopen_dirs, &argv, &self.vars)?;
    |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 5 parameters

error: aborting due to 2 previous errors
```

--- a/src/commands/run.rs
+++ b/src/commands/run.rs
@@ -17,8 +17,7 @@ use wasmtime_interface_types::ModuleData;
 use wasmtime_wasi::{
     create_wasi_instance, old::snapshot_0::create_wasi_instance as create_wasi_instance_snapshot_0,
 };
-#[cfg(feature = "wasi-c")]
-use wasmtime_wasi_c::instantiate_wasi_c;
+
 #[cfg(feature = "wasi-c")]
 use wasmtime_wasi_c::instantiate_wasi_c;
 
@@ -152,7 +151,8 @@ impl RunCommand {
             #[cfg(feature = "wasi-c")]
             {
                 let global_exports = store.global_exports().clone();
-                let handle = instantiate_wasi_c(global_exports, &preopen_dirs, &argv, &self.vars)?;
+                let handle =
+                    instantiate_wasi_c("", global_exports, &preopen_dirs, &argv, &self.vars)?;
                 Instance::from_handle(&store, handle)
             }
             #[cfg(not(feature = "wasi-c"))]

